src/infra/src/table/short_urls.rs

```diff
@@ -194,8 +182,7 @@ pub async fn contains(short_id: &str) -> Result<bool, errors::Error> {
         .filter(Column::ShortId.eq(short_id))
         .into_model::<ShortUrlRecord>()
         .one(client)
-        .await
-        .map_err(|e| Error::DbError(DbError::SeaORMError(e.to_string())))?;
+        .await?;
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.